diagnostic.md

```diff
@@ -184,5 +201,13 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+30 px
```

Comment:
At 550px screen width, the `@media(min-width= 400px)` takes over. Thus `.stretch` class will have a width of 25%. So 25% * 550px = 137.5px.